README.md

```diff
@@ -1,102 +0,0 @@
-# Timed Application
```

Comment:
Can you replace this with a very quick and dirty small README that mainly links the chart instead of deleting it?